shared_libraries/orca_shared/database/adapters/api/aws.py

```diff
@@ -1,15 +1,18 @@
 import json
-import logging
 import os
 
 import boto3
+from aws_lambda_powertools import Logger
 
 from orca_shared.database.entities.postgres_connection_info import (
     PostgresConnectionInfo,
 )
 
+# Set AWS powertools logger
+Logger = Logger()
 
-def get_configuration(db_connect_info_secret_arn: str, logger: logging.Logger) \
+
+def get_configuration(db_connect_info_secret_arn: str, logger: Logger) \
```

Comment:
Revert to old type-hint. This code doesn't need to know about the powertools logger since we are passing it in.

Reply:
revered